func/omp/header_api_support.cpp

```diff
@@ -0,0 +1,38 @@
+#include <omp.h>
+#include <stdio.h>
+#include <faasm/faasm.h>
+
+bool failed = false;
+
+struct ThreadInfo {
+  int32_t thread_num;
+  int32_t local_thread_num;
+} __attribute__ ((aligned(64)));
```

Comment:
Is the `aligned(64)` necessary? Not necessarily a problem, just interested

Reply:
Technically no, I just want to avoid false sharing when the threads are writing back to the same struct. Now that I think of it I think I meant `128`

Reply:
This will be moved to a global set of macros however like Clang does it